roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_rhel_kvm/assert-configuration.yml

```diff
@@ -72,13 +79,13 @@
         success_msg: "PASS: kvm.nx_huge_pages is {{ sap_hypervisor_node_preconfigure_nx_huge_pages }}"
       ignore_errors: "{{ sap_hypervisor_node_preconfigure_ignore_failed_assertion }}"
 
-  when: sap_hypervisor_node_preconfigure_nx_huge_pages is defined
-
 - name: Assert - check seccomp_sanbox=0
   block:
-    - command: grep -E '^seccomp_sandbox\s+=\s+0.*$' /etc/libvirt/qemu.conf
+    - name: Get seccomp setting
+      ansible.builtin.command: grep -E '^seccomp_sandbox\s+=\s+0.*$' /etc/libvirt/qemu.conf
```

Comment:
Same, use lookup and regex filter?

Reply:
I would like to leave the rhel_kvm code as it is for now, but I'll keep that in my backlog.